vulnfeeds/cmd/debian-copyright-mirror/debian-copyright-mirror.py

```diff
@@ -0,0 +1,163 @@
+"""
+Download all of the copyright files for packages in main in Debian unstable.
+
+This:
+
+Parses https://metadata.ftp-master.debian.org/changelogs/filelist.yaml.xz 
+    to identify the files to retrieve
+Generates a curl configuration to download the URLs
+Executes curl to download the URLs in parallel
+"""
+
+import os
+import argparse
+import yaml
```

Comment:
We need to install [PyYAML](https://pypi.org/project/PyYAML/) in the docker container for this.

It's probably overkill to make a whole poetry environment for this & install poetry in the Dockerfile just for one dependency. Maybe we could just make a requirements file (with hashes) like [we have to install poetry](https://github.com/google/osv.dev/blob/911a80f78eaa496c62a1eda08a76d78521c65c08/docker/poetry/requirements.in).

And hm, `gcr.io/google.com/cloudsdktool/google-cloud-cli:485.0.0-alpine` doesn't seem to have `pip` installed, (but it does have `venv`). Not really sure what the best course of action there is.